tests/unit/store/api_service.spec.js

```diff
@@ -1,54 +1,34 @@
 import Vue from "vue";
-import axios from "axios";
-import VueAxios from "vue-axios";
-
 import { get_recording } from "@/store/ApiService.js";
 
-describe("api to get_recording", () => {
+describe("APIService", () => {
   beforeEach(() => {
     // jest.resetModules()
     // jest.clearAllMocks()
     jest.restoreAllMocks();
   });
 
-  it("Calls axios.get with the correct endpoint URL and returns the result of the HTTP request", async () => {
+  test("Given the endpoint URL is valid, When Axios does a get method call which is mocked, Then assert the value matches the mocked json object", async () => {
```

Comment:
mocking a method is always something that should be described in the "Given" section of a test, not the "When".  Mocking something is part of setup/arranging the environment to the point where you are ready to trigger the action you want to test